dozer/cogs/roles.py

```diff
@@ -0,0 +1,178 @@
+import discord, discord.utils
+from discord.ext.commands import bot_has_permissions, has_permissions, BadArgument
+from sqlalchemy.orm.exc import MultipleResultsFound, NoResultFound
+from .. import db
+from ._utils import *
+
+class Roles(Cog):
```

Comment:
where is Cog defined?

Reply:
Cog is defined in `cogs/_utils.py`, and imported by `from ._utils import *`.

Reply:
explicit is better than implicit